Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Controller.mo

```diff
@@ -0,0 +1,2275 @@
+      iconTransformation(extent={{-140,-310},{-100,-270}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput TChiWatSupSet(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Chilled water supply temperature"
+    annotation (Placement(transformation(extent={{800,630},{840,670}}),
+        iconTransformation(extent={{100,260},{140,300}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yChiWatPum[nChiWatPum]
+  if have_heaChiWatPum
+    "Chilled water pump status setpoint"
+    annotation(Placement(transformation(extent={{800,500},{840,540}}),
+      iconTransformation(extent={{100,230},{140,270}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yChiPumSpe[nChiWatPum](
+    final min=0,
+    final max=1,
+    final unit="1") "Chilled water pump speed setpoint"
+    annotation (Placement(transformation(extent={{800,460},{840,500}}),
+      iconTransformation(extent={{100,200},{140,240}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yChiDem[nChi](
+    final quantity=fill("ElectricCurrent",nChi),
+    final unit=fill("A", nChi)) if need_reduceChillerDemand
+    "Chiller demand setpoint to set through BACnet or similar "
+    annotation(Placement(transformation(extent={{800,400},{840,440}}),
+      iconTransformation(extent={{100,170},{140,210}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yChi[nChi]
+    "Chiller enabling status setpoint"
+    annotation(Placement(transformation(extent={{800,330},{840,370}}),
+      iconTransformation(extent={{100,140},{140,180}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yHeaPreConValSta[nChi]
+    "Chiller head pressure control status setpoint"
+    annotation (Placement(transformation(extent={{800,260},{840,300}}),
+      iconTransformation(extent={{100,110},{140,150}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yHeaPreConVal[nChi](
+    final min=fill(0, nChi),
+    final max=fill(1, nChi),
+    final unit=fill("1", nChi))
+    "Head pressure control valve position"
+    annotation (Placement(transformation(extent={{800,220},{840,260}}),
+      iconTransformation(extent={{100,70},{140,110}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yConWatPumSpe[nConWatPum](
+    final unit=fill("1", nChi),
+    final min=fill(0, nChi),
+    final max=fill(1, nChi))
+    "Condenser water pump speed setpoint"
+    annotation (Placement(transformation(extent={{800,130},{840,170}}),
+        iconTransformation(extent={{100,40},{140,80}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yChiWatMinFloSet(
+    final quantity="VolumeFlowRate",
+    final unit="m3/s",
+    final min=0) "Chilled water minimum flow setpoint"
+    annotation (Placement(transformation(extent={{800,100},{840,140}}),
+      iconTransformation(extent={{100,12},{140,52}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yConWatPum[nConWatPum]
+    "Status setpoint of condenser water pump"
```

Comment:
"Status setpoint" is not clear: "on/off command" or "start signal" is clearer.

"Setpoint" should be reserved for the ***desired value of a controlled variable***.